modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeSetResultsSetTotalMethodCheck.java

```diff
@@ -43,6 +44,40 @@ protected String[] getValidExtensions() {
 		return new String[] {"java", "jsp", "jspf"};
 	}
 
+	private String _addTryCatch(
+		String newContent, String methodCall, String stringBuilder) {
+
+		newContent = StringUtil.replace(newContent, methodCall, stringBuilder);
+
+		Matcher matcher = _setResultsAndTotalPattern.matcher(newContent);
+
+		String newMethodCall = null;
+		StringBuilder sb = new StringBuilder();
+
+		if (matcher.find()) {
+			String tabs = SourceUtil.getIndent(matcher.group());
+			String newLine = StringPool.NEW_LINE + StringPool.TAB;
+
+			newMethodCall = matcher.group();
```

Comment:
@regiCesar21 in this method the `matcher.group()` call its repeated, we could move the newMethodCall to the if beginning and reuse to the getIdent call, What do you think?